x-pack/plugins/siem/public/components/loading/index.tsx

```diff
@@ -44,10 +48,17 @@ export const LoadingPanel = pure<LoadingProps>(
     >
       <LoadingStaticContentPanel>
         <EuiPanel className={showBorder ? '' : 'euiPanel-loading-hide-border'}>
-          <EuiLoadingChart size="m" />
-          <EuiText>
-            <p>{text}</p>
-          </EuiText>
+          <EuiFlexGroup alignItems="center" direction="row" gutterSize="none">
+            <SpinnerFlexItem grow={false}>
+              <EuiLoadingSpinner size="m" />
+            </SpinnerFlexItem>
+
+            <EuiFlexItem grow={false}>
+              <EuiText>
+                <p>{text}</p>
```

Comment:
Looking at the demo's: 
https://elastic.github.io/eui/#/display/text

I think EuiText already adds a `<p>` and this is just extra DOM weight:

From demo
<img width="1323" alt="Screen Shot 2019-05-01 at 11 45 11 AM" src="https://user-images.githubusercontent.com/1151048/57032054-9c560f00-6c06-11e9-82f6-fd5d0124f860.png">